extensions/color-casket/src/validators.ts

```diff
@@ -0,0 +1,7 @@
+import { isValidHSL, isValidRGB, isValidColorName } from "is-valid-css-color";
+
+function isValidHEX(color: string) {
+  return /^#([A-Fa-f0-9]{3}([A-Fa-f0-9]{3})?)$/.test(color);
```

Comment:
just my regex OCD suggestion 😆 
```suggestion
  return /^#([A-Fa-f0-9]{3}){1,2}$/.test(color);
```